### ISSUE TITLE
Accessibility of the Login page

### ISSUE THREAD

**Issue by bl-aire** [jupyterhub/jupyterhub#4260]
### Details from report:
Based on evaluation using the WAVE accessibility tool, the following are responsible for accessibility issues on the Login page:

- **Language missing or invalid**: Currently, the language attribute `lang="en"` which helps search engines switch language profiles so they can provide correct accent/pronunciation is missing in the opening html tag `<html>` .

- **Very low contrast**: The contrast of the form's header background colour, its button background colour and their text is too low. The same applies to the error message displayed when a user's password or username are not correct. The background colours of affected elements can be made darker to fix this so their text is legible for anyone with low vision or colour deficiencies.

- **Missing first level header**: This accessibility issue is as a result of the page having no `h1` element. 
The form's header was created with this code `<div>Sign in</div>`. I would propose nesting a `h1` in the `div` and reducing the font-size and possibly the font-weight of the header to fix it.

- **Positive tabindex values present**: 
Tabindex values of 1 or greater places the element the tabindex is applied first in the keyboard tabbing order. Thus, creating a confusing experience for anyone who relies on the keyboard’s Tab key to navigate.
Interactive elements come with native browser behavior already built in that allows them to be navigated to via Tab without the need for a tabindex declaration. In this case, the elements are in the reading order of the document so the Tabindex is not necessary.

- **Noscript element**

**Suggestion:** The alt text of the logo is "jupyterhub". I am suggesting that it is renamed to "jupyterhub logo".  This would give users an idea of what the image's purpose is.

### Report:

<img width="1280" alt="Screenshot 2022-12-08 at 13 43 16" src="https://user-images.githubusercontent.com/84882370/206695138-a0534019-306c-451e-9cdd-ec856243628d.png">

Hello @minrk,
As discussed, I have evaluated and given a summary of the login page. I am open to a discussion on how best they can be fixed.

**Comment by minrk** [comment]
> Currently, the language attribute lang="en" 

Yes, this would be in `page.html` and fix every page. We should make sure it can be overridden, in case deployments localize their pages in other languages. We can do this with a template variable, I think.

>  The alt text of the logo is "jupyterhub". I am suggesting that it is renamed to "jupyterhub logo"

👍 

For the h1 question (which will come up for all pages), I asked the accessibility team about that a while back https://github.com/jupyter/accessibility/discussions/88. Their recommendations there will be useful. I think making the `Sign in` heading an h1, and then adding CSS to keep it a sensible size makes sense.

I think we can just remove the tabindex, but we should make sure that the focus starts at the username input.

The color is a trickier question, and one I think we should involve accessibility/design folks in. Because just darkening the orange means we're deviating from Jupyter branding, so we might need to think of another design altogether.

**Comment by bl-aire** [comment]
I have not made a comment about Noscript element because I am still researching on it. When I have better understanding of it and know how to fix it, I will propose a solution.

**Comment by minrk** [comment]
Yes, I suspect the recommendation may be to _not_ change the color, following this discussion about the exact same color combination: https://github.com/jupyter/jupyter.github.io/issues/587 . We can save it for now.

**Comment by bl-aire** [comment]
> Yes, I suspect the recommendation may be to _not_ change the color, following this discussion about the exact same color combination: [jupyter/jupyter.github.io#587](https://github.com/jupyter/jupyter.github.io/issues/587) . We can save it for now, at least.

Okay, noted
